myokit/formats/mathml/_parser.py

```diff
@@ -947,3 +951,13 @@ def _parse_name(self, element):
         except Exception as e:
             raise MathMLError('Unable to create Name: ' + str(e), element)
 
+    def _parse_symbol(self, element):
```

Comment:
The `csymbol` was previously not supported by the MathML parser right? Your first suggestion would be really cool in general, but maybe the latter one, just updating the docstring, would make more sense in this PR? So if you ever needed full `csymbol` support one could think about how to generalise this better. 
Or in other words, do you think you would want to add `csymbol` for other importers any time soon, and it would therefore make sense to worry about this now?

Reply:
OK go for the docstring for now and I'll make a separate issue